app/models/metric/targets.rb

```diff
@@ -70,14 +70,10 @@ def self.capture_storage_targets(hosts)
     hosts.flat_map(&:storages).uniq.select { |s| Storage.supports?(s.store_type) & s.perf_capture_enabled? }
   end
 
-  def self.capture_vm_targets(targets, parent_class, options)
-    enabled_parents = targets.select do |t|
-      t.kind_of?(parent_class) &&
-        t.kind_of?(Metric::CiMixin) &&
-        t.perf_capture_enabled? &&
-        t.respond_to?(:vms)
```

Comment:
Nice @kbrock, yeah, I have no idea why these checks were here other than maybe the intention was to allow this for non-Host types at some point.  I"m fine with putting this back if we ever want to do it that way.  The new way seems simpler to me.